aspnetcore/tutorials/signalr-typescript-webpack/sample/3.x/src/index.ts

```diff
@@ -11,19 +11,19 @@ const connection = new signalR.HubConnectionBuilder()
     .build();
 
 connection.on("messageReceived", (username: string, message: string) => {
-    let messageContainer = document.createElement("div");
+    let m = document.createElement("div");
```

Comment:
```suggestion
    let messages = document.createElement("div");
```

Stick with a more descriptive variable name to provide clues around the `div` element's purpose. I've used `messages` in my recommendation, in case you want something shorter. The original variable name would be even more descriptive.

Reply:
I updated the script per your suggestion and tested.  Thanks.